main.nf

```diff
@@ -73,14 +73,27 @@ if (params.help) {
 */
 
 include { SEQQC } from './workflows/seqqc'
-
+include { ANNOTATION } from './workflows/genome_annotation'
```

Comment:
I don't see this workflow file as part of the PR. Is the plan to create it during the hackathon?

Reply:
Yes, that will be created during the demo in the hackathon. Also, that will stay on the `28-workflow-for-genome-annotation` branch till we close the issue #28 (linked to several other issues)